zoo_project.py

```diff
@@ -74,24 +77,88 @@ def __repr__(self):
 
 
 class Animal:
-    """Adds an animal to a cage, required attributes are:
-    animal_species
-    animal_name
-    cage
+
+    """Creates an animal
+    name
     """
 
-    def __init__(self, animal_name, animal_species):
-        self.animal_name = animal_name
-        self.animal_species = animal_species
+    def __init__(self, name):
+        self.name = name
+
+    def __repr__(self):
+        return "{}".format(self.name)
+
+
+class Lion(Animal):
+
+    """Species class will define the specific characteristics of the animals that are not shared between them
+    """
+
+    def __init__(self, name):
+        Animal.__init__(self, name)
+        self.species = "Lion"
         # If the animal species has known preys, they will be added as the prey property of the animal,
         #  to be used when the animal is added to the cage.
-        if animal_species.lower() == "lion":
-            self.preys = ["sheep", "wildebeest", "impala", "zebra", "giraffe", "buffalo", "wild hog", "rhinoceros",
-                          "hippopotamus"]
-        if animal_species.lower() == "tiger":
-            self.preys = ["boar", "wild pig", "bear", "buffalo", "wild cattle", "deer", "antelopes", "monkey"]
-        if animal_species.lower() == "hyena":
-            self.preys = ["impalas", "gazelles", "wildebeests", "zebras", "goat", "sheep", "horse"]
+        self.preys = ["sheep", "wildebeest", "impala", "zebra", "giraffe", "buffalo", "wild hog", "rhinoceros", 
+                      "hippopotamus"]
+
+    def __repr__(self):
+        return "{} the {}".format(self.name, self.species)
+
+
+class Tiger(Animal):
+
+    """Species class will define the specific characteristics of the animals that are not shared between them
+    """
+
+    def __init__(self, name):
+        Animal.__init__(self, name)
+        self.species = "Lion"
+        # If the animal species has known preys, they will be added as the prey property of the animal,
+        #  to be used when the animal is added to the cage.
+        self.preys = ["impalas", "gazelles", "wildebeests", "zebras", "goat", "sheep", "horse"]
+
+    def __repr__(self):
```

Comment:
This is repeated in a few places and is not DRY.  Can you work out how to define it in a single place?